deploy/042_deploy_lisk_adapter.ts

```diff
@@ -20,10 +20,6 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
       L1_ADDRESS_MAP[chainId].liskCrossDomainMessenger,
       L1_ADDRESS_MAP[chainId].liskStandardBridge,
       L1_ADDRESS_MAP[chainId].usdc,
-      // L1_ADDRESS_MAP[chainId].cctpTokenMessenger,
-      // For now, we are not using the CCTP bridge and can disable by setting
-      // the cctpTokenMessenger to the zero address.
-      ZERO_ADDRESS,
```

Comment:
Might be worth proactively applying this change to the Blast and Mode deployment scripts as well :+1:

Reply:
Good call. Done here: https://github.com/across-protocol/contracts/pull/541/commits/3b8d0d7fe4e7a5a19547eea6f64070ac0b15c612